test/core/security/grpc_tls_crl_provider_test.cc

```diff
@@ -64,24 +66,43 @@ namespace testing {
 class FakeDirectoryReader : public DirectoryReader {
  public:
   ~FakeDirectoryReader() override = default;
-  absl::StatusOr<std::vector<std::string>> GetFilesInDirectory() override {
-    if (return_bad_status_) {
-      return absl::UnknownError("");
-    }
+  absl::StatusOr<std::vector<std::string>> GetDirectoryContents() override {
     return files_in_directory_;
   }
+  absl::string_view Name() override { return kCrlDirectory; }
 
-  void SetReturnBadStatus(bool value) { return_bad_status_ = value; }
-  void SetFilesToReturn(std::vector<std::string> files) {
+  void SetFilesInDirectory(absl::StatusOr<std::vector<std::string>> files) {
     files_in_directory_ = std::move(files);
   }
+  void SetName(absl::string_view value) {
```

Comment:
I don't think we need this, or the `directory_path_` data member.

Reply:
Good catch, I had originally made it settable, but there was no time where I didn't just want it to be the crl directory